application/models/Question.php

```diff
@@ -86,8 +86,10 @@ class Question extends LSActiveRecord
     /** @var string $group_name Stock the active group_name for questions list filtering */
     public $group_name;
     public $gid;
-    /** Set defaut relevance **/
+    /** Defaut relevance **/
     public $relevance = '';
+    /** defaut same_script , avoid public break during update **/
+    public $same_script = 0;
```

Comment:
It's the fix , i don't understand your question …

If DB is updated : came from DB, if not : came from here if exist. Since it not exist in DB : not set before DB update (public access only)
Then : public access broke before DB updated.